en/config/safety.md

```diff
@@ -0,0 +1,225 @@
+# Safety Configuration (Failsafes) 
+
+PX4 has configurable failsafe systems to protect and recover your vehicle if something goes wrong! These allow you to specify areas and conditions under which you can safely fly, and the [action](#failsafe_actions) that will be performed if a failsafe is triggered (for example, landing, holding position, or returning to a specified point).
+
+The most important failsafe settings are configured in the *QGroundControl* [Safety Setup](#qgc_safety_setup) page. Others must be configured via [parameters](#failsafe_other).
+
+## Failsafe Actions {#failsafe_actions}
+
+Each failsafe defines its own set of actions. Some of the more common failsafe actions are:
+
+Action | Description
+--- | --- 
+None/Disabled | No action (the failsafe will be ignored).
+Warning | A warning message will be sent to *QGroundControl*.
+Loiter | The vehicle will enter [Hold mode](../flight_modes/hold.md). For multicopters this means the vehicle will hover, while for fixed/wing the vehicle will circle.
+Return to Land | The vehicle will enter [Return mode](../flight_modes/return.md). Return behaviour can be set in the [Return Home Settings](#return_settings) (below).
+Land at current position |  The vehicle enters [Land mode](../flight_modes/land.md), and lands immediately.
+RC Auto Recovery (CASA Outback Challenge rules) | ?
+Terminate/Flight termination | ?
```

Comment:
What happens in Terminate/Flight termination. Is this just "Kill -drop out of the sky?"

Reply:
PS There were notes in geofence for this:

> Note: Setting this value to 4 enables flight termination, which will kill the vehicle on violation of the fence. Due to the inherent danger of this, this function is disabled using a software circuit breaker, which needs to be reset to 0 to really shut down the system. 

So it does sound like this is a kill switch. If so, 
- what is the "Breaker" (ie another parameter). 
- What happens instead of killing if you don't have the breaker enabled?
- Does this same breaker apply to any termination, or just due to Geofence?

Reply:
- Terminate: the effect is a mode switch in commander, which turns off all controllers, and more importantly for a user, all outputs are set to a failsafe pwm value. The failsafe value can be defined in an airframe config (`FAILSAFE ` variable).
- Lockdown: bypasses everything, and just kills the motors (sets them to disarmed). This is the same as using the kill switch.

For geofence: the circuit breaker param is CBRK_FLIGHTTERM, and it's only used when flight termination is requested from navigator (i.e. geofence violation).